### DISCUSSION TITLE
我需要将打包好的文件部署到 aws s3 或者ali oss，如何做路由映射

### DISCUSSION THREAD

**Discussion by wuyoooo** [umijs/umi#9791] [General]
我需要将打包好的文件部署到 aws s3 或者ali oss 。但是我刷新 /a/b 这样的路由地址，最终会得到一个404。如何将路由映射到/index.htm.有过成熟的方法吗？

**Comment by fz6m** [comment]
在搜索引擎上找一下 spa 前端应用的重写 index.html 方法。

比如 nginx ，需要 try_files 。
